Functions/Administration/xASL_adm_GeneratePdfConfig.m

```diff
@@ -0,0 +1,140 @@
+function [config] = xASL_adm_GeneratePdfConfig(x, subject, bOverwrite)
```

Comment:
Would rename all QC-related functions to xASL_qc_ and move to the QualityControl folder, just easier to have these all sorted.